### DISCUSSION TITLE
Add custom attribute above 255

### DISCUSSION THREAD

**Discussion by gk-fschubert** [radsecproxy/radsecproxy#125] [Q&A]
Hi, 

is there a option to add a custom attribute using the rewrite block with a number higher than 255? 
In FreeRADIUS you have the option to add custom attributes but should only use id's above 3000 to not interupt other vendors. 

Or may I'm just not understanding something correctly. 
Currently the server just quits without any information why.
![image](https://github.com/radsecproxy/radsecproxy/assets/41149338/a27293af-4aef-4db1-8e5e-be37f2feac0a)

**Comment by fmauchle** [comment]
Radius attributes have a fundamental attribute-id space of 8 bits (255 attribute types).

What you likely want (and FreeRADIUS is referring to) is the Vendor-Specific Attribute (id 26). It is specified as a 24 bit vendor id ([IANA enterprise number](https://www.iana.org/assignments/enterprise-numbers/)), plus 8 bit vendor attribute id (subattribute). 

radsecproxy does support rewrite for vendor-specific attribute using the `AddVendorAttribute vendor:subattribute:value` (and similar) config.
